web/client/codechecker_client/cmd_line_client.py

```diff
@@ -974,13 +1105,13 @@ def print_reports(client, reports, output_format):
                 else:
                     output.append(report)
             print(CmdLineOutputEncoder().encode(output))
-            return
 
-        if output_format == 'html':
-            output_dir = args.export_dir
-            if not os.path.exists(output_dir):
-                os.makedirs(output_dir)
+            if len(output_formats) == 1:
```

Comment:
Just mentioning that `# Json was the only format specified.` above this would help the reader tremendously.